### PR TITLE
Add CITEseq part to Besca

### PR DESCRIPTION
**Pull request by ansonrel**
- added DSB normalization function through rpy2. It is now an option to use this normalization or CLR (default) in the notebook. Parameter to be set by the user at the beginning of the workbook. 
- added a script for the installation of required R packages for the DSB wrapper to work (conda/pip installation cannot install all R packages unfortunately). I placed it in the `besca` folder but maybe there's a better location for it ...?
- Extended the CITEseq part of the standard_workflow notebook. It is now possible to compare the clusters computed from proteins (UMAP or riverplots), visualize the raw vs normalized signals and more. 
- The celltype_annotation notebook can now use any combination of RNA/CLR/DSB-based clustering to annotate the cells. This can lead to some differences of annotation, mainly for scarce cell-types. Same for UMAP-representation: the user can now view the cells in a UMAP computed on the RNA, CLR or DSB values. These parameters can be changed in the first chunks of the notebook. 
- Changed the accessor functions of Kotliarov and Granja datasets so that both return an `adata` of protein counts. 
- None of the above changes should affect the default RNA workflow.

### CODE REVIEW

**Comment by ajulienla** [comment]
Hi @ansonrel ,
thank you for the great work.
I have two comments open to discussion:

- I think the Rlibs might be better located at the root of the package ? (with requirement.txt?) I do not know what is the common practices there.  What do you think? 
- Should we have a wrapper_R.py file to keep well separated the two for now ? Would that make sense ? Not from a functional  point of view but from a development one ? We discussed to extend this with additional functions so this might increase and having separated files pure python and rpy2 might help our development ? What do you think?

**Comment by ansonrel** [comment]
Thanks for your answer. Concerning your 2 comments:
1) Unfortunately, I'm also unaware of the common practices in mixed Python/R modules ... Maybe someone else has an opinion on it and else, I will move it at the root, yes. 
2) From our perspective, it's indeed better to have separated R and Python script I agree. The problem that I see is for the users to retrieve some functions, which are at the moment organized in `pp`, `tl`, etc. Maybe we could separate the R wrappers in 1 wrapper_R.py **per** submodule ? (maybe it's what you suggested in the first place?)

File: besca/Rlibs.R [hunk 1]
```diff
@@ -0,0 +1,17 @@
+args = commandArgs(trailingOnly=TRUE)
+libloc <-  args[1]
+
+dir.create(libloc)
+
+.libPaths(libloc)
+if (!require("devtools")) install.packages("pacman", lib = libloc)
```
Comment (ajulienla) [inline besca/Rlibs.R:7]: Some copy paste issue? 

if (!require("devtools")) install.packages(**"devtools"**, lib = libloc)

Same from **withr** and **Matrix** ?

Comment (ansonrel) [reply besca/Rlibs.R:7]: Wops, yes! my bad, sorry.

**Review by hatjek — APPROVED** [review]
Hi @ansonrel ! Thank you for all the improvements! I tested the standard workflow notebook on the citeseq branch in a fresh Python-only conda environment and also the DSB normalization in a Python/R conda environment. Both work smoothly.

**Pull request merged**